site/2015/03/31/doctrine_mongo_odm_module_release_0_8_2.rst

```diff
@@ -0,0 +1,28 @@
+Doctrine Mongo ODM Module release 0.8.2
+====================================
```

Comment:
Note that in RST, you should have the same amount of `=` as the length of the title